src/shared/constants/wordClass.js

```diff
@@ -0,0 +1,62 @@
+export default {
+  ADJ: {
+    label: 'Adjective',
+    description: 'Adjective',
```

Comment:
Now that I think about this structure, I think it would make more sense to have a `value` field that holds the key value.

For example: 
```
ADJ: {
    value: 'ADJ',
    label: 'Adjective',
}
```

That way when someone indexes into this object, they would be able to get the tagset value along with the label. On top of that, it seems like all the `label` and `description` values are the same so the `description` key will no longer be necessary.

Reply:
Ok, great... I'll fix asap

Reply:
Fixed